app/views/articles/_video_player.html.erb

```diff
@@ -5,27 +5,27 @@
     <meta itemprop="description" content="<%= @article.description %>" />
     <meta itemprop="thumbnailUrl" content="<%= cloudinary(article.video_thumbnail_url, 880) %>" />
     <meta itemprop="contentUrl" content="<%= article.video_source_url %>" />
-    <meta itemprop="duration" content="<%= "PT%sM%sS" % article.video_duration_in_minutes.split(':') %>" />
+    <meta itemprop="duration" content="<%= "PT%sM%sS" % article.video_duration_in_minutes.split(":") %>" />
```

Comment:
The issue can be fixed by:
```erb
<% minutes, seconds = @article.video_duration_in_minutes.split(":") %>
<meta itemprop="duration" content="<%= format("PT%<minutes>sM%<seconds>sS", minutes: minutes, seconds: seconds) %>" />
```
If anyone has an idea how to fix it prettier, please share.
@Glennmen please, also pull from master to get the new `erblint` config, or it'll complain on "unused variables".